### ISSUE TITLE
Manual physical monitor rotation broken in 244

### ISSUE THREAD

**Issue by riggles1** [libretro/mame#313]
Using RA 1.10.3 MAME 244
Before 244 I would set the rotation off in the core options and set all the rotation options to 0 in the mame.ini, no overrides or saves/mame config files active. Every new vertical game would be displayed sideways on a horizontal screen as default.

I'm using a cab with a rotating monitor, so whenever I play a vertical game I will just rotate it manually.
But now it'll not play vertical fullscreen/sideways/non-rotated no matter what unless I change individual game settings and have CW90 for these games. Booting new games without this cfg makes my CRT go interlaced with the vertical game on the "horizontal screen" with black bars.

In short, ever since 244 there's no way to have the auto-rotation of games disabled, my only workaround right now is to enter the MAME menu and set rotation to CW90, since even the .ini isn't read for this (other .ini options are read just fine).

Hoping this can be fixed, last time this was broken you at least had the workaround of setting everything regarding core rotation to 0 in the mame.ini.

Thanks!

**Comment by riggles1** [comment]
Still broken as of 245, this bug results in audio stuttering when you set Namco Classics Collection Vol 1+2 and set their saves/mame/.cfg's to   <target index="0" rotate="90" /> (because this core isn't ignoring rotation like it should).

Why these two games cause stuttering on a CRT setup? Related to an issue with incorrect refresh reporting of these games (from main mame) where the machine has 58.55Hz but the game is set to 60.61Hz
The only MAME machine with this incorrect refresh set that I've encountered.

This would all be a non issue if rotation worked like it did before 244 where you could block the core from rotating the screen through the mame.ini, for manual CRT rotation setups.


RA core options "allow rotation" set to off.

MAME set to read from ini in the config/.opt

Within mame.ini;

#
# CORE ROTATION OPTIONS
#
rotate                    0
ror                       0
rol                       0
autoror                   0
autorol                   0
flipx                     0
flipy                     0
